src/ergw_api.erl

```diff
@@ -51,3 +56,14 @@ collect_contexts(Context, Tunnels) ->
     io:format("Context: ~p~n", [Context]),
     Info = gtp_context:info(Context),
     [Info#{'Process' => Context} | Tunnels].
+
+shuffle(List) ->
+    [Y || {_, Y} <- lists:sort([ {rand:uniform(), El} || El <- List])].
+
+delete_random_peers([], Count) -> {error, Count};
+delete_random_peers(_, 0) -> ok;
+delete_random_peers([{_, Peer} | Tail], Count) when is_pid(Peer) ->
+    ok = gen_server:call(Peer, {unbind, self()}),
```

Comment:
* a `unbind` will not terminate/remove/shutdown a path,
* **it is absolute horrible style and not acceptable to do a direct gen_sever call/cast into another module, never ever do that without having a really, really, really good reason**